docs/settings/alert-action-settings.asciidoc

```diff
@@ -577,12 +577,6 @@ For a <<servicenow-action-type,{sn-itsm}>>, <<servicenow-sir-action-type,{sn-sir
 [[alert-settings]]
 === Alerting settings
 
-`xpack.alerting.maxEphemeralActionsPerAlert` {ess-icon}::
-deprecated:[8.8.0]
-Sets the number of actions that will run ephemerally. To use this, enable
-ephemeral tasks in task manager first with
-<<task-manager-settings,`xpack.task_manager.ephemeral_tasks.enabled`>>
```

Comment:
hmmm ... not sure.  I know the allow-list must be used when creating / updating the Kibana yml in the clould admin panels, but when else would it be used?  If it gets used at some other time (cluster restart), then potentially it could prevent the restart because of invalid config?

It's fine if it's only used while creating / updating the Kibana yml in the cloud admin panel - presumably the config wouldn't be saveable next time a customer edited it, till they removed the config setting.

Have we done this in the past?  I feel like we must have ...

Reply:
I guess that's true. I guess we're just ignoring it now if it's set so it doesn't matter if it's still there

Reply:
Perhaps we should create a PR to label these in comments as deprecated, if we're not sure?  Breadcrumbs for future cleanups?

Reply:
From https://github.com/elastic/cloud/pull/133738#discussion_r1828211350, we can set a `max` value as we have to keep it around for deployments running 8.x.